.gitignore

```diff
@@ -85,6 +85,7 @@ celerybeat-schedule
 # virtualenv
 venv/
 ENV/
+.venvwrapper/
```

Comment:
Did you commit it by mistake? Why don't you change the name of your .venvwrapper to venv? or you can do a local gitignore.